app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -305,6 +305,16 @@ class HomeFragment : Fragment(), AccountObserver {
         (activity as AppCompatActivity).supportActionBar?.hide()
 
         requireComponents.backgroundServices.accountManager.register(this, owner = this)
+
+        val closeAllPrivateTabs = arguments
+            ?.let(HomeFragmentArgs.Companion::fromBundle)
+            ?.let { it.closeAllPrivateTabs } ?: false
+
+        arguments = null
```

Comment:
Do the arguments need to be cleared out? Other parts of the fragment could use them.